src/formatDate.js

```diff
@@ -50,7 +50,55 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  // happy checking
+
+  const inputArray = date.split('');
+  const result = [];
+
+  for (let i = 0; i < inputArray.length; i++) {
+    if (isNaN(inputArray[i] / 2)) {
+      inputArray[i] = ' ';
+    }
+  }
```

Comment:
im not sure i understand what this is for. it looks like the way to remove the separator then you can simply pop it from the from format and split by that to get a clean array

Reply:
Okay fixed